app/views/about.html

```diff
@@ -1 +1,43 @@
-<p>This is the about view.</p>
+<div class="about">
+    <h1>Welcome to MoodCat.me</h1>
+    <p>
+        Hi there!
+        Welcome to MoodCat.me, the music platform of the future, available today.
+        Join us in the adventure of finding and enjoying music geared to you!
+        We try to understand you
+    </p>
+    <h2>How does it work?</h2>
+    <p>
+        Go to the homepage, and select what mood(s) you're in.
+        Once you click next, we recommend you a couple of rooms to check out.
+        Pick one you think you like, and you will join the room.
+        We'll start playing music for you, and you can chat with people in the same mood as you are!
```

Comment:
'Immediately music will start playing', we don't do anything, the system does. Also the comma before the and could be omitted.

'Immediately music will start playing and you can interact with people in the same mood.'